### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by imsandip2085**

### CODE REVIEW

File: src/Components/FeedbackMessageBox.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import { createdSettingRequest } from "../redux/Action/actions";
+import { useDispatch, useSelector } from 'react-redux';
+
+function FeedbackMessageBox({ handleCloseFeedback, enableFeedback }) {
+    const [feedbackMessage, setFeedBackMessage] = useState("")
+    const dispatch = useDispatch();
+
+    const handleMessage = (e) => {
+        console.log(e.target.value, "feedbackMessage");
```
Comment (manishiitg) [inline src/Components/FeedbackMessageBox.js:10]: remove

File: src/redux/Saga/SettingCreatedSaga.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { createdSettingSuccess, createdSettingError } from "../Action/actions";
+import { put, call } from "redux-saga/effects";
+import axiosCall from "../../Services/index";
+
+
+export function* SettingCreatedSaga(action) {
+    console.log(action.payload,"payloadpayload");
+    try {
+        const url = `https://us-central1-platoo-polls.cloudfunctions.net/settings/create`
```
Comment (manishiitg) [inline src/redux/Saga/SettingCreatedSaga.js:9]: why are api url's hard coded this should come from env everywhere? in the full app

Comment (imsandip2085) [reply src/redux/Saga/SettingCreatedSaga.js]: ok removing

File: src/Components/FeedbackMessageBox.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+function FeedbackMessageBox({ handleCloseFeedback, enableFeedback }) {
+    const [feedbackMessage, setFeedBackMessage] = useState("")
+    const dispatch = useDispatch();
+
+    const handleMessage = (e) => {
+        setFeedBackMessage(e.target.value)
+    }
+    const handleMessageSubmit = () => {
+        dispatch(createdSettingRequest({
+            "domain_name": "company1",
```
Comment (manishiitg) [inline src/Components/FeedbackMessageBox.js:14]: why is this harcoded

Comment (manishiitg) [reply src/Components/FeedbackMessageBox.js:14]: this is so much wrong. i cannot even say

Comment (manishiitg) [reply src/Components/FeedbackMessageBox.js:14]: is domain hard coded anywhere else at all?

File: src/Components/Home.js [hunk 4]
```diff
@@ -110,47 +120,12 @@ function PollListing(props) {
           deletePollDomain={home.deletePollDomain}
         />
         : null}
-        {/* <div className="pie-chart">
```
Comment (manishiitg) [inline src/Components/Home.js:113]: why is this chart removed?

File: src/Config/index.js [hunk 5]
```diff
@@ -0,0 +1 @@
+export const baseUrl = "https://us-central1-platoo-polls.cloudfunctions.net";
```
Comment (manishiitg) [inline src/Config/index.js:1]: all the other api's in admin, is the url hard coded for them?

File: src/redux/Saga/loginsaga.js [hunk 6]
```diff
@@ -1,6 +1,7 @@
 import { call, put } from "redux-saga/effects";
 import axiosCall from "../../Services/index";
 import { LogInSuccess, LogInError } from "../Action/actions";
+import { baseUrl } from '../../Config/index';
 
 function* LogInsaga(action) {
   const url = `https://us-central1-steady-datum-291915.cloudfunctions.net/Login`;
```
Comment (manishiitg) [inline src/redux/Saga/loginsaga.js:7]: what is this stupidity ?

File: src/Components/FeedbackMessageBox.js (hunk 3 above)
Comment (imsandip2085) [reply src/Components/FeedbackMessageBox.js:14]: it will come from localstorage

File: src/Components/Home.js (hunk 4 above)
Comment (imsandip2085) [reply src/Components/Home.js:113]: checking

File: src/redux/Saga/loginsaga.js (hunk 6 above)
Comment (imsandip2085) [reply src/redux/Saga/loginsaga.js:7]: it is already used at login time, now not use here, I forget to remove it

File: src/Config/index.js (hunk 5 above)
Comment (imsandip2085) [reply src/Config/index.js:1]: No, I have changed now

File: src/Components/Home.js (hunk 4 above)
Comment (imsandip2085) [reply src/Components/Home.js:113]: In starting, I have implemented a graph here, but here is not use this graph now

**Pull request merged**